web/beacon-app/src/features/members/components/MemberInfo.tsx

```diff
@@ -0,0 +1,60 @@
+import { Trans } from '@lingui/macro';
+
+import { SentryErrorBoundary } from '@/components/Error';
+import { formatDate } from '@/utils/formatDate';
+
+import type { MemberResponse } from '../types/memberServices';
+
+interface MemberDetailInfoProps {
+  data: MemberResponse;
+}
+
+const MemberDetailInfo: React.FC<MemberDetailInfoProps> = ({ data }) => {
+  const { id, name, role, created } = data || {};
+
+  return (
+    <SentryErrorBoundary
+      fallback={
+        <div className="item-center justify-center text-center text-sm text-danger">
+          <Trans>
+            Sorry, we were unable to load the user details. You can either refresh the page or get
```

Comment:
I didn't check if this comes from Figma, but should the support email address be included as it is in similar error messages?